test/cpp/api/tensor.cpp

```diff
@@ -17,7 +17,7 @@ bool exactly_equal(at::Tensor left, T right) {
 }
 
 template <typename T>
-bool almost_equal(at::Tensor left, T right, T tolerance = 1e-4) {
+bool almost_equal(at::Tensor left, T right, double tolerance = 1e-4) {
```

Comment:
why is this changed to `double`?

Reply:
Because we can not do
```C++
double < complex
```

Reply:
ahh I see for `c10::complex<U>` the tolerance would be `complex` otherwise